doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+accurate catalog available while taking into account the proper motions of the astrometric sources in the field based on
+the date the exposures were taken in the visit.
+
+By default, the most current GAIA catalog will
+be used as the default astrometric catalog for aligning all exposures which typically results in images that are aligned
+well enough to avoid serious misalignment artifacts in the regions of overlap. Unfortunately, due to the nature of some
+of the fields observed by HST, not all exposures can be aligned to a GAIA-based
+catalog using GAIA astrometric sources.  As a result, the sources used for alignment will have much larger errors on the
+sky compared to the errors of GAIA astrometric sources which can lead to offsets from the GAIA coordinate system of a
+significant fraction of an HST pixel (or even multiple HST pixels).  This will lead to artifacts in MVM products where
+such exposures overlap other exposures aligned to other catalogs, especially GAIA-based catalogs.
+
+
+Defining the Mosaic on the Sky
+==============================
+MVM products need to be defined in order to understand what input exposures will contribute to each mosaic.  The solution
+implemented for HAP MVM products relies on tesselation of the entire sky using the same basic tiles defined by the
+PanSTARRS project as described at `PanSTARRS Sky tessellation patterns
+<https://outerspace.stsci.edu/display/PANSTARRS/PS1+Sky+tessellation+patterns>`_.
+
+.. figure:: images/figure_aitoff.png
+  :figwidth: 90%
+  :alt: Aitoff plot of all 2,009 PS1 projection cells for the 3PI survey.
+
+  Aitoff plot of all 2,009 PS1 projection cells for the 3PI survey.  The coverage extends from declination −30° to the
+  north celestial pole.
+
+
+.. figure:: images/figure_pole.png
+  :figwidth: 90%
+  :alt: PS1 projection cells near the north celestial pole.
+
+  PS1 projection cells near the north celestial pole, where the image overlap is greatest due to convergence of the RA grid.
+  The projection cells are 4°x4° in size and are on rings spaced by 4° in declination.
+
+MVM processing uses these pre-defined projection cells after extending them to cover the entire sky
+with cells that are actually 4.2°x4.2° in size and are on rings spaced by 4° apart in declination.  This provides
+sufficient overlap to allow data from one projection cell to be merged with data from a neighboring cell if necessary.
+
+All these definitions, including finely each ProjectionCell should be divided to define the SkyCells, are encoded in a
+FITS table installed as part of the `drizzlepac` package:
+
+    drizzlepac/pars/allsky_cells.fits
+
+
+Defining each SkyCell
+----------------------
+Each projection cell is split into a grid of 21 x 21 'sky cells' which serves as the most basic MVM product generated
+during MVM processing.  Sky cells are approximately 0.2degrees x 0.2degrees in size (~21500 x ~21500 pixels) and
```

Comment:
Use degree symbol here instead of text for consistency with other text.

Reply:
Changed in 241a672